encryption/src/main/java/org/wildfly/security/encryption/CipherUtil.java

```diff
@@ -126,8 +128,13 @@ public static String decrypt(final String token, final SecretKey secretKey) thro
         Cipher cipher = Cipher.getInstance(TRANSFORMATION);
         AlgorithmParameterSpec spec = new IvParameterSpec(iv);
         cipher.init(Cipher.DECRYPT_MODE, secretKey, spec);
-
-        return cipher.doFinal(cipherText);
+        byte[] decryptedBytes;
+        try {
+            decryptedBytes = cipher.doFinal(cipherText);
+        } catch (IllegalBlockSizeException | BadPaddingException e) {
```

Comment:
These are both `GeneralSecurityException`s which this method already throws so I don't think this is needed.

Reply:
Removed